jenkins/shared-libraries/linux/vars/AnalyzeCodeQLDatabase.groovy

```diff
@@ -0,0 +1,72 @@
+def call(repo, language) {
+    env.DATABASE_BUNDLE = sprintf("%s-database.zip", language)
+    env.DATABASE_PATH = sprintf("%s-%s", repo, language)
+    if(!env.ENABLE_DEBUG) {
+        env.ENABLE_DEBUG = false
+    }
+    if(!env.ENABLE_CODEQL_DEBUG) {
+        env.ENABLE_CODEQL_DEBUG = false
+    }
+    env.LANGUAGE = language
+    if(env.LANGUAGE != "javascript" && env.LANGUAGE != "python" && env.LANGUAGE != "ruby") {
```

Comment:
This would be a bit easier than checking each:
```
["javascript", "python", "ruby"].contains(env.LANGUAGE)
```

Reply:
Thank you, much cleaner!